### PR TITLE
[FLINK-34892][ci] Fix python test failure due to config file change

### PR DESCRIPTION
**Pull request by z3d1k**
## Purpose of the change

Removing `pyflink_gateway_server.py` override after Flink 1.19.0 release to support configuration file change.
Override was initially introduced [here](https://github.com/apache/flink-connector-kafka/pull/69) and replicated for AWS connectors.

## Verifying this change

Please make sure both new and modified tests in this PR follows the conventions defined in our code quality guide: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

Local test run

## Significant changes
*(Please check any boxes [x] if the answer is "yes". You can first publish the PR and check them afterwards, for convenience.)*
- [ ] Dependencies have been added or upgraded
- [ ] Public API has been changed (Public API is any class annotated with `@Public(Evolving)`)
- [ ] Serializers have been changed
- [ ] New feature has been introduced
  - If yes, how is this documented? not applicable

### CODE REVIEW

File: flink-python/tox.ini
Comment (z3d1k) [inline flink-python/tox.ini:24]: The current fix does not work for python 3.7 because there is no 1.19 package released for this python version.
Since dependency on `apache-flink` python package is set as `apache-flink>=1.17.0` - only the latest available Flink version is used during testing.

File: flink-python/dev/integration_test.sh
Comment (z3d1k) [inline flink-python/dev/integration_test.sh:47]: Removed to use `pyflink_gateway_server.py` provided by pyflink instead of local copy.

**Review by dannycranmer — APPROVED** [review]

**Comment by dannycranmer** [comment]
Thanks @z3d1k. Waiting for actions to complete before merge

**Pull request merged**
